app/liquid_tags/gist_tag.rb

```diff
@@ -32,6 +32,10 @@ def build_uri(link)
   end
 
   def parse_link(link)
+    if link.nil?
+      raise StandardError, "You have not provided link to the Gist"
```

Comment:
To follow the wording of the existing errors maybe something like this would be closer?
```md
Invalid Gist link: You must provide a Gist link
```